CONTRIBUTING.md

```diff
@@ -1,16 +1,88 @@
-# Contributing guide
+# Contribution guide
 
 This contribution guide aims to help you with the optimal way to get involved with this project.
 
-Please contact us if you need help or have ideas to improve this guide.
+## Table of contents
 
-## Content contributions (market data)
+* [Market data contributions](#market-data-contributions)
+* [Code contributions](#code-contributions)
+* [How to use Git](#how-to-use-git)
+* [General advices](#general-advices)
+
+Please do not hesitate to contact us if you need help or have ideas on how to improve this guide.
+
+
+## Market data contributions
+
+You can add market data for your city in a few steps. All you need to do is to **add one file** (_city.json_)
+and to **modify two existing** ones (_cities.json_, _README.md_). Here is how you start:
+
+![city.json][icon-city-json] ![cities.json][icon-cities-json] ![README.md][icon-readme-md]
```

Comment:
I am trying to make the documentation more accessible for new people. I think the icons help with getting the broad idea on how to add market data. I am very visual oriented person. I felt only text is very boring and "dry".